zeppelin-web/package-lock.json

```diff
@@ -3669,910 +3668,6 @@
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8=",
       "dev": true
     },
-    "fsevents": {
```

Comment:
Is this package-lock.json change related to ZEPPELIN-1967 or it's something else?

Reply:
Thanks @Tagar -  this was indeed an unintended change that crept in by mistake. 

I have reversed the changes out of the PR now.